assignment.rb

```diff
@@ -0,0 +1,47 @@
+require_relative "event_record.rb"
+event = {add: "1", month_view: "2", date_view: "3", all_events: "4", delete: "5", exit: "6"}  
+loop do 
+  puts
+  puts "For Adding an event, Enter: 1"
+  puts "For Viewing a specific month's event(s), Enter: 2"
+  puts "For Viewing a specific date's event(s), Enter: 3"
+  puts "For Viewing all events, Enter: 4"
+  puts "For Deleting an event, Enter: 5"
+  puts "To Quit, Enter 6"
```

Comment:
move everything to methods and classes to use pure OOP approach.

Reply:
done